cmd/util/cmd/epochs/cmd/recover.go

```diff
@@ -69,8 +85,19 @@ func addGenerateRecoverEpochTxArgsCmdFlags() error {
 	generateRecoverEpochTxArgsCmd.Flags().Uint64Var(&flagNumViewsInEpoch, "epoch-length", 0, "length of each epoch measured in views")
 	generateRecoverEpochTxArgsCmd.Flags().Uint64Var(&flagNumViewsInStakingAuction, "epoch-staking-phase-length", 0, "length of the epoch staking phase measured in views")
 	generateRecoverEpochTxArgsCmd.Flags().Uint64Var(&flagEpochCounter, "epoch-counter", 0, "the epoch counter used to generate the root cluster block")
-
-	err := generateRecoverEpochTxArgsCmd.MarkFlagRequired("epoch-length")
+	generateRecoverEpochTxArgsCmd.Flags().Uint64Var(&flagTargetDuration, "epoch-timing-duration", 0, "the target duration of the epoch, in seconds")
+	generateRecoverEpochTxArgsCmd.Flags().Uint64Var(&flagTargetEndTime, "epoch-timing-end-time", 0, "the target end time for the epoch, specified in second-precision Unix time")
```

Comment:
Good point, I agree we can and should specify only one of these two quantities to the command. Let's use `epoch-timing-duration`, since the same value can be re-used (less manual changes to parameters). 

For context, these flags were taken from the spork bootstrapping utility. In that context, we do need to provide 2 data-points (duration and target end time). Since the exact start time of the spork is in the future and not known at that point in time, we need both duration and end time to fully specify the parameters for the Block Time Controller. 

Here, the precise start time of the recovery epoch is known (it's the target end time of the current epoch, which we can retrieve from the snapshot). 

(We need to make sure the `TargetEndTime` API method accounts for extensions for this to work. See [comment](https://github.com/onflow/flow-go/pull/6102#pullrequestreview-2166989320).)

Reply:
https://github.com/onflow/flow-go/pull/6156/commits/a0df4a205e1680c1e6146b652ec8d6701754351d